src/test/java/bt/CryptoBlackJackTest.java

```diff
@@ -0,0 +1,31 @@
+package bt;
+
+import bt.sample.Hello;
+import org.junit.Test;
+import signumj.entity.response.AT;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Simple testings for contracts
+ *
+ * @author tyvain
+ */
+public class SimpleContractTest extends BT {
```

Comment:
Class and filename mismatch? Also, please do not send your application CryptoBlackJack.java to this repository, create a new one for that. For instance, there are other applications on this repository https://github.com/blockplay-io/contracts, you can create a new one for yours.

Reply:
it's a commit / push error. I will solve it